specification/maps/data-plane/Microsoft.Maps/Weather/preview/1.0/weather.json

```diff
@@ -76,6 +76,13 @@
     }
   },
   "parameters": {
+    "Pollutants": {
+      "name": "pollutants",
+      "in": "query",
+      "type": "boolean",
+      "required": false,
+      "description": "Boolean value that returns detailed information about each pollutant. By default is True."
```

Comment:
@john35452: `"x-ms-parameter-location": "method"` is also needed, this is a method parameter right?